deploy/manifests/kip/overlays/mac/provider.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+cloud:
+  aws:
+    accessKeyID: ""
+    secretAccessKey: ""
+etcd:
+  internal:
+    dataDir: /opt/kip/data
+cells:
+  bootImageSpec:
+    owners: 689494258501
+    filters: name=elotl-kip-*
+    macfilters: name=elotl-kipmac-*
+  defaultInstanceType: "t3.nano"
+  defaultVolumeSize: "20G"
+  nametag: kip
+  standbyCells:
+    - instanceType: "mac1.metal"
+      count: 1
+      spot: false
```

Comment:
We should rethink if we want to use standby instances at this point. The way they work, kip will allocate and start a new one if a pod gets bound to an existing one (basically headroom). This is good for reducing wait times, but while testing, it'll create wastage.

Reply:
:+1:  removing for now